sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/ReleaseDownloadListener.java

```diff
@@ -140,6 +141,7 @@ public boolean onComplete(@NonNull Uri localUri) {
     @Override
     public void onError(@NonNull String errorMessage) {
         AppCenterLog.error(LOG_TAG, errorMessage);
+        Toast.makeText(mContext, errorMessage, Toast.LENGTH_SHORT).show();
```

Comment:
Are we going to toast programmatic messages that are not in translatable resources?